src/nvim/generators/gen_api_dispatch.lua

```diff
@@ -119,56 +119,59 @@ for _,f in ipairs(shallowcopy(functions)) do
   if newname ~= nil then
     if function_names[newname] then
       -- duplicate
-      print("Function "..f.name.." has deprecated alias\n"
-            ..newname.." which has a separate implementation.\n"..
-            "Please remove it from src/nvim/api/dispatch_deprecated.lua")
```

Comment:
an example in non-text code of the same issue. the only thing wrong here is a hanging `..` which should be on the next line. Ideally it should be able to fix this formatting issue without completely _deformatting_ the entire statement at the same time.